pitch.md

```diff
@@ -1,20 +1,86 @@
+and people being extra conservative with respect to their availability) are
+addressed by allowing for greater expressiveness in describing one's schedule.
+This is because allowing for a gradation of availability (from totally free, to
+somewhat available, to completely busy) might make people less likely to
+game the system by saying that they are completely at times that are just
+suboptimal for them.  It also means that there is more information which a
+clever algorithm could make of when deciding which of two conflicts is worse.
+
+If I was able to help them, users would be able to specify, in a way that feels
+natural, their availability and preferences.  Hopefully this can reduce 
+the work of an individual (who must compile the results of an email) or the
+repetition inherent in painting over the same boxes for pairs of weeks when
+using a WhenIsGood&#8482;.
+
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+The essential gap I am trying to bridge is that of language.  I hope to provide
+users with a better way of expressing their schedules, to satisfy two specific
+goals:
+ * Less redundancy: No need to paint the same pattern over multiple weeks.
+ * More detailed information: Can specify a range of availability rather than 
+ a binary variable for each time slot.
+
+A simple calendar or WhenIsGood does not allow for this level of
+expressiveness. An user actually writing text (such as in the email scenario)
+does allow for arbitrarily high expressivenes, but would then shift the burden
+of interpreting users' availability onto another user (the organizer).  My
+language hopes to reach in the middle, allowing greater expressiveness than
+other common scheduling tools, without shifting the burden of interpreting
+this extra information onto the user.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+As I mentioned during the previous version of this assignment, I came
+up with the idea when I needed to fill out many of these schedules
+for all the different organizations whose meetings I attend. 
+
+At least for my schedule, I found that there were many patterns
+which would have been easier to describe than painting the same
+pattern of boxes over several different days.  There are also some 
+relationships that are not just difficult, but actually impossible to 
+currently express, such as variability preference of how well a 
+certain time fits, or the "I can make any of these hours work, but 
+not all of them at the same time." situation.
+
+I am excited about trying to make the language integrate with existing calendar
+formats, since this would involve finding a clever abstraction for all the
+information which could serve as a medium between the natural / intuitive
+language with which users would describe their schedule, and the highly
+structured data of an exported Google Calendar or iCal instance.
```

Comment:
This sounds exciting too&mdash;however, I would reserve this as a secondary or tertiary project goal and prefer expressiveness over interface.